crates/re_renderer/Cargo.toml

```diff
@@ -94,6 +94,14 @@ zip = { workspace = true, default-features = false, features = ["deflate"] }
 # For tests:
 unindent.workspace = true
 
+
+
+
+
+
+
+
+
```

Comment:
What happened here?

Reply:
I made sure `taplo fmt` is called whenever CI runs `crates.py version`